plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/AbstractTestOracleTypeMapping.java

```diff
@@ -893,11 +890,11 @@ private void runTestsWithSetup(DataSetup dataSetup, DataTypeTest... tests)
     }
 
     /**
-     * Run a {@link DataTypeTest} in the given time zone, using non-legacy timestamps.
+     * Run a {@link SqlDataTypeTest} in the given time zone, using non-legacy timestamps.
```

Comment:
> using non-legacy timestamps

this is obsolete too, but let's ignore this for now
instead of fixing the method javadoc, i'd rather remove (inline) it,
like in `io.trino.plugin.postgresql.TestPostgreSqlTypeMapping#testDate`